pkg/kando/location.go

```diff
@@ -20,14 +20,22 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	"github.com/kanisterio/kanister/pkg/datamover"
 	"github.com/kanisterio/kanister/pkg/param"
 )
 
 const (
-	pathFlagName    = "path"
-	profileFlagName = "profile"
+	pathFlagName             = "path"
+	profileFlagName          = "profile"
+	repositoryServerFlagName = "repository-server"
 )
 
+type DataMoverVal struct {
```

Comment:
What does `Val` mean here? Also, I think we should rename it `DataMoverArgs` what do you think?

Reply:
Done !